src/app/page.tsx

```diff
@@ -23,34 +24,28 @@ export default function Home() {
 function DibbsSection() {
   return (
     <ContentContainer align>
-      <div className="grid grid-cols-1 justify-items-center gap-4 xl:grid-cols-[1fr_2fr] xl:justify-items-start xl:gap-14">
-        <div className="order-1">
-          <Image
-            className="xl:max-h-[20rem] xl:max-w-[30rem]"
-            src={`${basePath}/images/placeholder.png`}
-            width={480}
-            height={320}
-            alt="Placeholder"
-          />
-        </div>
-        <div className="order-2 flex flex-col gap-5">
-          <h2 className="min-w-full text-center text-[1.5rem] font-bold text-[#224a58] xl:text-left xl:text-[2rem]">
-            Introducing Data Integration Building Blocks
-          </h2>
-          <div className="flex flex-col gap-2">
-            <p className="m-0 max-w-[39.7rem] p-0 text-base font-normal leading-relaxed text-[#224a58]">
-              {homeContent.dibbs.description}
-            </p>
-            <ul className="text-base font-semibold leading-relaxed text-[#224a58]">
-              {homeContent.dibbs.benefits.map((benefit, index) => (
-                <li className="min-w-full" key={`benefit-${index}`}>
-                  {benefit}
-                </li>
-              ))}
-            </ul>
-          </div>
+      <ImageCard
+        imageUrl={`${basePath}/images/homepage-1.jpeg`}
```

Comment:
This image appears to be cropped in the design (left) vs the implementation (right). Do we want to try to tackle that?
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/94f5fd71-4585-44ee-9e4d-292a2b1f86e2">

Reply:
<img width="1702" alt="image" src="https://github.com/user-attachments/assets/a249f72f-3291-4e35-8798-dba790a21374">